templates/author/completed.tpl

```diff
@@ -9,18 +9,19 @@
  *
  *}
 <div id="submissions">
-<table class="listing" width="100%">
-	<tr><td class="headseparator" colspan="{if $statViews}7{else}6{/if}">&nbsp;</td></tr>
+<table class="table table-condensed">
```

Comment:
The class table is a bootstrap element :O http://www.w3schools.com/bootstrap/bootstrap_tables.asp